### PR TITLE
Remove trunk params

### LINKED ISSUE #2: [Task3]: Remove the "TRUNK_PARAMS" from the VISSL configuration
**Task**: Currently, in VISSL, we have a key called "TRUNK_PARAMS" in the configuration files https://github.com/facebookresearch/vissl/blob/master/vissl/config/defaults.yaml#L360. We want to simplify the configuration file by removing the key from the config file. This will help reduce the user errors and also ensure a more intuitive use of VISSL.

**Steps**:
1. You can start by simply removing the key from the config file. Once you remove the key, you need to move this part of config https://github.com/facebookresearch/vissl/blob/master/vissl/config/defaults.yaml#L361-L385 unnested i.e. this was earlier nested under "TRUNK_PARAMS" and not it should be simply nested in "TRUNK"
2. Make the changes in all the config files under https://github.com/facebookresearch/vissl/tree/master/configs/config
3. Make the changes in the https://github.com/facebookresearch/vissl/tree/master/vissl/models/trunks https://github.com/facebookresearch/vissl/search?l=Python&q=TRUNK_PARAMS 
4. Make the changes in the tests https://github.com/facebookresearch/vissl/tree/master/tests https://github.com/facebookresearch/vissl/search?l=Python&q=TRUNK_PARAMS
5. Update the docs https://github.com/facebookresearch/vissl/tree/master/docs/source
6. Run all the CPU tests again
7. Submit the PR to VISSL and tag Priya for review.

### PR DESCRIPTION
**Pull request by grace-omotoso**
Resolves #2

### CODE REVIEW

File: vissl/trainer/train_zero_task.py
Comment (prigoyal) [inline vissl/trainer/train_zero_task.py]: nit: is this change intended? :)

File: tools/cluster_features_and_label.py
Comment (prigoyal) [inline tools/cluster_features_and_label.py]: nit: is this change intended? or is this in master?

File: docs/source/vissl_modules/models.rst
Comment (prigoyal) [inline docs/source/vissl_modules/models.rst]: this needs to be nested under `REGNET`

File: docs/source/vissl_modules/models.rst
Comment (prigoyal) [inline docs/source/vissl_modules/models.rst]: this needs to be nested under RESNET

File: docs/source/conf.py
Comment (prigoyal) [inline docs/source/conf.py]: nit: this change should be reverted! it will break vissl docs :)

File: .circleci/config.yml
Comment (prigoyal) [inline .circleci/config.yml]: quick check: are the circleCI changes expected in this PR?

**Review by prigoyal — CHANGES_REQUESTED** [review]
looks really great :) left some inline comments and good to go after those are addressed!

@uhbuhb I think it will be good time to upstream MLH to main vissl master as any configs that land in main master will then be able to land with the new structural change done in this PR :)

**Comment by uhbuhb** [comment]
> looks really great :) left some inline comments and good to go after those are addressed!
> 
> @uhbuhb I think it will be good time to upstream MLH to main vissl master as any configs that land in main master will then be able to land with the new structural change done in this PR :)

do you mean before this PR is merged or after? Ill open a PR for now...

**Comment by prigoyal** [comment]
@

> > looks really great :) left some inline comments and good to go after those are addressed!
> > @uhbuhb I think it will be good time to upstream MLH to main vissl master as any configs that land in main master will then be able to land with the new structural change done in this PR :)
> 
> do you mean before this PR is merged or after? Ill open a PR for now...

@uhbuhb , sorry about vague comment earlier. I think once this PR lands we can upstream this individual PR to main VISSL master. Then we can ensure that any new config files (yaml files) that come in VISSL are aligned with the PR changes.

File: .circleci/config.yml
Comment (uhbuhb) [reply .circleci/config.yml]: sorry, this was my fault.. Committed it to master without realising people will be adopting it to their feature branches. fixed :)

File: tools/cluster_features_and_label.py
Comment (grace-omotoso) [reply tools/cluster_features_and_label.py]: @prigoyal , this looks like some changes autogenerated by the isort command. I am looking into it

File: docs/source/vissl_modules/models.rst
Comment (grace-omotoso) [reply docs/source/vissl_modules/models.rst]: @prigoyal, that is a good catch. Thank you.

File: docs/source/vissl_modules/models.rst
Comment (grace-omotoso) [reply docs/source/vissl_modules/models.rst]: good catch, thanks

File: vissl/trainer/train_zero_task.py
Comment (akainth015) [reply vissl/trainer/train_zero_task.py]: This change looks like it was introduced by `isort` to enforce PEP 8 two newlines

File: tools/cluster_features_and_label.py
Comment (akainth015) [reply tools/cluster_features_and_label.py]: I think that is just `isort` fixing files that were previously non-conformant (alphabetical ordering of imports)

File: .circleci/config.yml
Comment (prigoyal) [inline .circleci/config.yml]: can we look into the gpus changes? are these needed?

File: .pre-commit-config.yaml
Comment (prigoyal) [inline .pre-commit-config.yaml]: coming from a previous PR?

**Review by prigoyal — CHANGES_REQUESTED** [review]

File: docs/source/conf.py
Comment (akainth015) [reply docs/source/conf.py]: I think this still needs to be addressed?

**Review by akainth015 — APPROVED** [review]
LGTM, aside from `docs/source/conf.py` I did not see any formatting changes.

File: docs/source/conf.py
Comment (grace-omotoso) [reply docs/source/conf.py]: I think that came through the previous commits from cherry-picking.  It is fixed now.

**Review by prigoyal — APPROVED** [review]
AWESOME work!!! thank you @grace-omotoso and @akainth015 :)

**Pull request merged**
